components/EpisodeTools.vue

```diff
@@ -147,16 +145,6 @@ const toggleShare = (event) => {
     event_label: 'Share Menu',
   })
 }
-
-// toggle function for toggle transcript emit
-const toggleTranscript = () => {
-  emit('toggleTranscript')
-  $analytics.sendEvent('click_tracking', {
```

Comment:
do we not want to track this anymore?

Reply:
It will track the link now that it's a real link